generators/docker-compose/templates/realm-config/jhipster-realm.json.ejs

```diff
@@ -2492,7 +2492,7 @@
     "clientOfflineSessionIdleTimeout": "0",
     "cibaInterval": "5"
   },
-  "keycloakVersion": "18.0.0",
+  "keycloakVersion": "19.0.0",
```

Comment:
I hope you diffed the new realm configurations to identify the version as the only change.

You also need to update the `generators/server/templates/src/main/docker/config/realm-config/jhipster-realm.json.ejs` with similar change

Reply:
Thank you so much Vishal,  please correct me if my understanding is not correct, actually 

```generators/server/templates/src/main/docker/config/realm-config/jhipster-realm.json.ejs```
 is resolved via  DOCKER_KEYCLOAK_VERSION constants

```/generators/generator-constants.js ```

![image](https://user-images.githubusercontent.com/534679/183287740-42998c20-69b7-4eb6-a4b9-482dad7f40c5.png)

Reply:
ok. In that case, you just need to check for diff to be included in the change